Dockerfile

```diff
@@ -1,38 +1,14 @@
-FROM ubuntu:14.04
-MAINTAINER Colin Alston <colin@praekelt.com>
-RUN apt-get update && apt-get -y --force-yes install libjpeg-dev zlib1g-dev libxslt1-dev libpq-dev nginx redis-server supervisor python-dev python-pip
-RUN apt-get -y install libffi-dev gettext
+FROM praekeltfoundation/django-bootstrap:onbuild
+RUN apt-get-install.sh git libjpeg-dev zlib1g-dev libffi-dev gettext libtiff-dev
 
-RUN pip install --upgrade pip
-
-ENV PROJECT_ROOT /deploy/
+ENV PROJECT_ROOT /app/
 ENV DJANGO_SETTINGS_MODULE tuneme.settings.docker
+ENV APP_MODULE "tuneme.wsgi:application"
 
-WORKDIR /deploy/
-
-COPY tuneme /deploy/tuneme
-COPY polls /deploy/polls
-COPY locale /deploy/locale
-ADD manage.py /deploy/
-ADD requirements.txt /deploy/
-ADD setup.py /deploy/
-ADD README.rst /deploy/
-ADD VERSION /deploy/
-
-RUN pip install -e .
-
-RUN mkdir -p /etc/supervisor/conf.d/
-RUN mkdir -p /var/log/supervisor
-RUN rm /etc/nginx/sites-enabled/default
-
-ADD docker/nginx.conf /etc/nginx/sites-enabled/molo.conf
-ADD docker/supervisor.conf /etc/supervisor/conf.d/molo.conf
-ADD docker/supervisord.conf /etc/supervisord.conf
-ADD docker/docker-start.sh /deploy/
-ADD docker/settings.py /deploy/tuneme/settings/docker.py
-
-RUN chmod a+x /deploy/docker-start.sh
-
-EXPOSE 80
+ENV CELERY_APP tuneme
+ENV CELERY_BEAT 1
```

Comment:
If you're doing a bunch of related `ENV` statements in a row, it's better to do the form that allows multiple declarations in one statement. Each `ENV` statement creates a new image layer and we want to reduce those.
You can do:
```dockerfile
ENV CELERY_APP=tuneme \
    CELERY_BEAT=1
```